viewing_party/party.py

```diff
@@ -1,23 +1,180 @@
 # ------------- WAVE 1 --------------------
-
+#adding a comment in the code to commit :)
 def create_movie(title, genre, rating):
-    pass
+    
+    user_data = {
+        "watched": [],
+        "watchlist": []
+        }
+
+    new_movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    if None in new_movie.values():
+        return None

```

Comment:
Nice validation check!

Additionally, It's an even better idea to run checks like this before we take up memory with a possibly unnecessary variable -- we could move this check above setting your `new_movie` dictionary and return None if title or genre or movie == None.